spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java

```diff
@@ -109,42 +109,26 @@ static void sleep(int seconds) {
     }
 
     /*
-    static Runnable moveToFrontLater = new Runnable() {
-        @Override
-        public void run() {
-            sleep(5);
-            SwingUtilities.invokeLater(moveToFront);
-        }
+    static Runnable moveToFront = () -> {
```

Comment:
I don't see why it is needed to modify commented out code.

Reply:
The modification is not required but is consistent with the modifications of similar methods.